resty/src/App.test.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
-import { render } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+import React from 'react'
+import { render, fireEvent, waitFor, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import App from '../app-number.js'
+import People from '../people'
+
+test('dynamically updates number', async () => {
+  render(<App />);
+  const input = screen.getByTestId('num');
+  const h1 = screen.getByTestId('output');
+  fireEvent.change(input, { target: { value: 1234 } });
+  expect(h1).toHaveTextContent('1234')
+})
+
+it('should render Star Wars list', async () => {
+
+  const people = {
+    "Luke Skywalker": "http://swapi.dev/api/people/1/",
+    "Darth Vader": "http://swapi.dev/api/people/4/",
+  }
+
+  render(<Results loading={false} results={results} />);
+
+  // screen.debug();
+
+  // Starting point for a testing approach
+  const items = screen.getByText('count: ');
```

Comment:
You have items here, and you have it down below as well now that is causing errors as well